src/plugins/intel_gna/gna_graph_compiler.cpp

```diff
@@ -598,6 +619,8 @@ void GNAGraphCompiler::finalizeConvolution2DPrimitive(InferenceEngine::CNNLayerP
     if (convolution._padding_y != convolution._pads_end.at(Y_AXIS)) {
         THROW_GNA_LAYER_EXCEPTION(layer) << "Convolution's input padding is not symetric along Y axis";
     }
+    convolution._padding_x = convolution._pads_end.at(X_AXIS);
+    convolution._padding_y = convolution._pads_end.at(Y_AXIS);
```

Comment:
just wonder if we should change content of convolution if we still can throw exception- shouldn't we in such case leave it untouched?